plugins/main/public/components/common/welcome/agent-info/agent-info.tsx

```diff
@@ -117,10 +117,10 @@ export class AgentInfo extends Component<AgentInfoProps> {
           value: agent,
           label: 'Operating system',
           render: (value: Agent) => (
-            <>
+            <div style={{ display: 'flex', alignItems: 'center', gap: 5 }}>
               {getPlatformIcon(value)}
               {getOsName(value)}
-            </>
+            </div>
```

Comment:
See https://oui.opensearch.org/1.19/#/layout/flex#flex-items-are-also-flex

```suggestion
            <EuiFlexGroup alignItems="center" gutterSize="none" style={{ gap: 5 }}>
                <EuiFlexItem>
                    <div>
                        {getPlatformIcon(value)}
                    </div>
                </EuiFlexItem>
                <EuiFlexItem>
                    {getOsName(value)}
                </EuiFlexItem>
            </EuiFlexGroup>
```